src/main/kotlin/sliv/tool/model/visualization/Landmark.kt

```diff
@@ -0,0 +1,4 @@
+package sliv.tool.model.visualization
+
+abstract class Landmark(val uid: Long, val layer: Layer) {
```

Comment:
why are models not separated by modules? firstly separate by modules then by mvc arch